components/gitpod-cli/cmd/open.go

```diff
@@ -57,12 +57,22 @@ var openCmd = &cobra.Command{
 
 		if wait {
 			pargs = append(pargs, "--wait")
+			ctx = cmd.Context()
```

Comment:
@mustard-mh looking at the code again, it seems fine to use the timeout-enabled context everywhere besides the final call to the editor binary, since we want to be able to time out on the other operations that are not expected to run potentially indefinitely.

Running the following inside the `gitpod-cli` directory seems to behave as expected, even when taking more than 10s to close the file:
```bash
go run . open --wait main.go 
```